ui/src/main/webapp/src/app/executions/executions-list.component.ts

```diff
@@ -70,14 +79,37 @@ export class ExecutionsListComponent implements OnInit {
     }
 
     canCancel(execution: WindupExecution): boolean {
-        return execution.state === 'QUEUED'; // || execution.state === 'STARTED';
+        return execution.state === 'QUEUED' || execution.state === 'STARTED';
```

Comment:
I'm not sure if I haven't missed anything. Does it support cancellation of running execution?